components/tfx/Transform/component.py

```diff
@@ -4,16 +4,15 @@
 
 
 def Transform(
-    input_data_path: InputPath('Examples'),
-    #examples: InputPath('Examples'),
+    examples_path: InputPath('Examples'),
     schema_path: InputPath('Schema'),
 
-    transform_output_path: OutputPath('TransformGraph'),
-    #transform_graph_path: OutputPath('TransformGraph'),
+    transform_graph_path: OutputPath('TransformGraph'),
     transformed_examples_path: OutputPath('Examples'),
```

Comment:
this one can be merged with examples_path

Reply:
All of these parameters correspond to the parameters of TFX components.
`examples_path` points to the source examples while the `transformed_examples_path` is for the transformed examples.